src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -224,16 +224,35 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
     }
 
+    void TermControl::SearchNextMatch()
+    {
+        if (!_searchBox)

```

Comment:
Shouldn't we load the searchbox?

Reply:
Nah. If they get here without a searchbox, how would they have a Match? :smile:

Reply:
No wait this is a good idea - if they execute these actions and the box hasn't been loaded, we probably should load it.